baseapp/middleware.go

```diff
@@ -90,3 +101,34 @@ func AccessHandler(f AccessCallback) func(next http.Handler) http.Handler {
 		})
 	}
 }
+
+var DefaultOtelFilter = filters.None(
+	filters.PathPrefix("/ping"),
+	filters.PathPrefix("/api/ping"),
+	filters.PathPrefix("/health"),
+	filters.PathPrefix("/api/health"),
+	filters.PathPrefix("/debug"),
+	filters.PathPrefix("/api/debug"),
+	filters.PathPrefix("/pprof"),
+	filters.PathPrefix("/api/pprof"),
+)
+
+func NewOtelHandler() func(http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			requestID := ""
+			if xid, ok := hlog.IDFromRequest(r); ok {
+				requestID = xid.String()
+			}
+
+			h := othttp.NewHandler(next, r.URL.Hostname(),
```

Comment:
I think `r.Host` is the right value here, I believe request URLs have only a path.

Reply:
Does the host come without protocol?

Reply:
Yes, `r.Host` should be the value of the `Host` header for the request